core/db/curibio/alembic/versions/49fceab10078_create_pulse3d_tables.py

```diff
@@ -0,0 +1,131 @@
+"""create pulse3d tables
+
+Revision ID: 49fceab10078
+Revises: 1fbfdf12dfbb
+Create Date: 2022-04-15 08:08:25.056643
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql as pg
+from sqlalchemy.sql import func
+
+revision = "49fceab10078"
+down_revision = "1fbfdf12dfbb"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
there's already an uploads table that I added, we might want to use that instead or delete it if we go with this one
https://github.com/CuriBio/k8s_curi/blob/main/core/db/curibio/alembic/versions/258ca806ee8b_initial_jobs_queue_table.py#L27